constructor/nsis/_nsis.py

```diff
@@ -296,6 +296,16 @@ def add_to_path(pyversion, arch):
     broadcast_environment_settings_change()
 
 
+def add_condabin_to_path():
+    from _system_path import (
+        add_to_system_path,
+        broadcast_environment_settings_change,
+    )
+
+    add_to_system_path(os.path.normpath(os.path.join(ROOT_PREFIX, "condabin")), allusers)
```

Comment:
I see in `constructor/nsis/main.nsi.tmpl` that addings condabin to path is disabled for alluser installs to address CVE-2022-26526. Should this piece of code also protect against this? Like:

```
if allusers:
  # log error, throw error?
else:
  add_to_system_path(os.path.normpath(os.path.join(ROOT_PREFIX, "condabin")), allusers)

Reply:
Thanks for spotting this!